x-pack/plugins/ingest_manager/public/applications/ingest_manager/sections/agent_config/list_page/index.tsx

```diff
@@ -331,7 +336,7 @@ export const AgentConfigListPage: React.FunctionComponent<{}> = () => {
             />
           </h2>
         }
-        actions={createAgentConfigButton}
+        actions={false ?? createAgentConfigButton}
```

Comment:
not sure to understand this expression it's not always going to return `createAgentConfigButton`?

Reply:
Didn't mean to add that. Will fix